app/models/patient.rb

```diff
@@ -188,6 +188,10 @@ def call_result=(new_call_result)
     super(new_call_result)
   end
 
+  def prescribed_drugs(date: Date.today)
```

Comment:
```suggestion
  def prescribed_drugs(date: Date.current)
```

Reply:
Could we add some trivial test coverage for this method too? Ensure that the `patient` invokes the right method on its `prescription_drugs` with the right arguments?